docs/certs.md

```diff
@@ -0,0 +1,12 @@
+---
+Certification management
+---
+
+![Certification management diagram](./assets/certs.svg)
+
```

Comment:
Some more comments on the diagram:
1. The font color in the third diamond from the left seems inconsistent with the rest of them.
2. The questions in the last block (Expiration) are incorrect. Remove `does` from the last two diamonds. How about rephrasing them to `cert from (...) secret expires within a given time?`?
3. Remove `is` from the fourth diamond and `no` from the first two diamonds.
4. You use `CA bundle` in the diagram and `caBundle` in the description. Shouldn't they be the same?
5. In the top left-hand corner of the diagram there's `self signed`. Change it to `self-signed`.
6. If there are questions, there should be some answers. How about adding yes/no arrows?